### DISCUSSION TITLE
PortableApps version and installed version not playing well together

### DISCUSSION THREAD

**Discussion by sgjohnson1981** [freeplane/freeplane#873] [Help] [answered]
I decided to use the PortableApps version to test out alpha previews so as not to mess with the main install on the machine. I logon to Windows 10 under the admin account, but some of the files/apps I use remain on a non-admin account and I use shortcuts ran through runas.exe to run them. For some reason I can open the PortableApps version residing in the admin account first, then open the "installed" version which I use through a non-admin account via runas.exe, but I can't go in the opposite direction. If I try it tells me I already have a version opened. Is there something I can change to get that to work? It's a bit annoying to have to close the installed version, then reopen it.

**Comment by EdoFro** [comment]
> Is there something I can change to get that to work?

test this idea:

uncheck this option in preferences for both Freeplane installations:
![image](https://user-images.githubusercontent.com/35700575/206569864-05cfc5b1-8482-4582-9742-2d486c02725e.png)

Please report back if it helped.

**Reply by sgjohnson1981** [reply]
Already unchecked on both versions.

**Comment by dpolivaev** [comment, marked as answer]
Portable Freeplane is developed to be used together with portable launcher, and usually it should not depend on the account.

Different major change versions like 1.10.x and 1.11.x usually can be used and installed simultaneously because they do not share any configuration files.

**Reply by sgjohnson1981** [reply]
Not sure what you're saying. I just downloaded Freeplane Portable from the SourceForge site and put a java installation into the FreeplanePortable folder, and changed the config file to point to the java folder as "java home." Are you saying if I used the main PortableApps program it should work?

**Reply by dpolivaev** [reply]
PortableFreeplane is created to be used together with the PortableApps.com Platform from https://portableapps.com/download

**Reply by dpolivaev** [reply]
If you do not use the platform you should generally avoid using portable freeplane and use the standard installer or the zipped distribution

**Reply by sgjohnson1981** [reply]
Tried using the platform. Made no difference.

**Reply by dpolivaev** [reply]
If the portable apps version does not work for you, why couldn't you use just the zip or windows installer version?

**Reply by sgjohnson1981** [reply]
Fantastic!! The freeplane_bin*.zip version doesn't cause the error. I was unaware there was another standalone version besides the PortableApps version. As far as I've tried the windows installer version, it doesn't allow to change the installation folder if it finds a version already installed. If there's a way to do that, I don't know how. As I said above, I'm just trying to not install a preview version over my stable version already installed.

**Comment by EdoFro** [comment]
maybe you can try these:
1. change the name of the freeplane folder inside Portable Apps,
    - for example from "FreeplanePortable" to "FreeplanePortable_A"
2. install the second version of Freeplane also as a portable app

I've done that before and it works. But I really don't remember if I worked with different Freeplane versions at the same time.

**Reply by EdoFro** [reply]
this was using the PortableApps launcher

**Reply by sgjohnson1981** [reply]
Not sure what you're saying in regards to step 2. Are you saying to install Freeplane into the PortableApps platform rather than a standalone? If so, just tried it. Made no difference. Step 1 made no difference either.

**Reply by EdoFro** [reply]
I say that you can install multiple versions of Freeplane in Portable Apps. You just have to change the folder name of the already installed version so the new one doesn't overwrite it.

**Reply by sgjohnson1981** [reply]
Dimitry's answer above will work. I've had the stable version installed for so long I don't really want to tamper with it.
